packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -572,22 +606,28 @@ export class LitContracts {
   };
 
   public static async getStakingContract(
-    network: 'cayenne' | 'manzano' | 'habanero' | 'custom' | 'localhost',
+    network: LIT_NETWORKS_KEYS,
     context?: LitContractContext | LitContractResolverContext,
     rpcUrl?: string
   ) {
     let provider: ethers.providers.JsonRpcProvider;
-    rpcUrl = rpcUrl ?? DEFAULT_READ_RPC;
+    rpcUrl =
+      rpcUrl ?? network === 'datil-dev' ? LIT_RPC.VESUVIUS : LIT_RPC.CHRONICLE;
```

Comment:
I think this is a bug. The following yields different result, and which is what (I think) we want instead.

```js
    rpcUrl =
      rpcUrl ??
      (network === 'datil-dev' ? LIT_RPC.VESUVIUS : LIT_RPC.CHRONICLE);
```